hack/lib/test.sh

```diff
@@ -154,3 +154,20 @@ hyper::test::execvm() {
   test $res == aaa
   sudo hyperctl rm $id
 }
+
+# regression test for #542 and #537
+hyper::test::remove_container_with_volume() {
+  mkdir -p  ${HYPER_TEMP}/tmp
+  sed -e "s|TMPDIR|${HYPER_TEMP}/tmp|" ${HYPER_ROOT}/hack/pods/single-volume.pod > ${HYPER_TEMP}/single-volume.pod
```

Comment:
should be `simple-volume.pod` instead of `single-volume.pod`, or you should change the file name